DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: neo4r
 Title: A 'Neo4J' Driver
-Version: 0.1.3
+Version: 4.0.0
 Authors@R: 
     c(person(given = "Colin",
```

Comment:
@davidlrosenblum @Slapphomet As you're both been working on this, could you add yourself as contributors here?

On the model of https://github.com/ThinkR-open/golem/blob/dev/DESCRIPTION#L31 for example